CONTRIBUTING.md

```diff
@@ -34,38 +36,45 @@ Status of open issues:
 - `(no label)`: The default status.
 - `contribution welcome`: The fix or enhancement is approved and you are invited to contribute to it.
 
+_It may happen that we add more labels to individual issues. These are used to describe the issue more granularly, but you don't need to worry about that._
+
 ## Contribute Code
 
 You are welcome to contribute code to `ui5-webcomponents-react` in order to fix issues or to add new features.
 
 ## Developer Certificate of Origin (DCO)
 
-Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this projects, this happens in an automated fashion during the submission process. 
+Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this projects, this happens in an automated fashion during the submission process.
 SAP uses [the standard DCO text of the Linux Foundation](https://developercertificate.org/).
 
 ### How to contribute - the Process
 
-1.  Make sure the issue you've filed in the [issue tracker](https://github.com/SAP/ui5-webcomponents-react/issues) has the label "contribution welcome" - otherwise, it is not ready to be worked on.
-2. Fork the `ui5-webcomponents-react` repository into your GitHub account, create a branch and apply your change. Please don't work directly on the `master` branch of your fork. This will help to update your fork in the future more easily.
-3. Commit and push your change on that branch.
-      Please follow our [Development Conventions and Guidelines](/docs/Guidelines.md).
-4. Create a pull request from your branch into `ui5-webcomponents-react`-`master` branch.
-5. Follow the link posted by the CLA assistant to your pull request and accept it, as described above.
-6. Wait for our code review and approval, possibly enhancing your change on request.
-      Note that the `ui5-webcomponents-react` developers have many duties. So, depending on the required effort for reviewing, testing, and clarification, this may take a while.
-7. Once the change has been approved and merged, we will inform you in a comment.
-8. Celebrate! 🎉
+1. Please let us know, that you want to work on the issue you've filed in the [issue tracker](https://github.com/SAP/ui5-webcomponents-react/issues).
+2. If the issue is marked with the `contribution welcome` tag, we'll assign you right away, otherwise we will check if some more information our guidance is needed and only then assign you to it.
+3. Once you are assigned to the issue, you are ready to go.
+4. Fork the `ui5-webcomponents-react` repository into your GitHub account, create a branch and apply your change. Please don't work directly on the `main` or `master` branch of your fork. This will help to update your fork in the future more easily.
```

Comment:
```suggestion
4. Fork the `ui5-webcomponents-react` repository into your GitHub account, create a branch and apply your change. Please don't work directly on the `main` branch of your fork. This will help to update your fork in the future more easily.
```